game/src/main/scala/io/aigar/game/Resources.scala

```diff
@@ -2,10 +2,28 @@ package io.aigar.game
 
 import io.aigar.game.serializable.Position
 
+object Resources {
+  final val MaxRegular = 200
+  final val MaxSilver = 100
+  final val MaxGold = 50
+}
+
 class Resources {
-  var regular = List(new Position(12, 12))
-  var silver = List(new Position(112, 112))
-  var gold = List(new Position(212, 212))
+  var regular = initRegular
+  var silver = initSilver
+  var gold = initGold
+
+  def initRegular: List[Position] ={
+    List.fill(Resources.MaxRegular)(Position(scala.util.Random.nextInt(900*3), scala.util.Random.nextInt(700*3)))
```

Comment:
How about using [`randomPosition` on `Grid`](https://github.com/DrPandemic/aigar.io/blob/master/game/src/main/scala/io/aigar/game/Grid.scala#L14)? It will automatically handle the dynamic dimensions of the map there!